src/components/HomepageBanner.js

```diff
@@ -0,0 +1,301 @@
+import React from 'react';
+import { css } from '@emotion/react';
+import { Button, Link } from '@newrelic/gatsby-theme-newrelic';
+import bannerForeground from '../images/bannerForeground.svg';
+import bannerForegroundMobile from '../images/bannerForegroundMobile.svg';
+import relicsAtWork from '../images/photo-relics-at-work-01.png';
+import relicsAtWorkMobile from '../images/relics-at-work--SM.png';
+
+const BREAKPOINTS = {
+  NO_LEFT_NAV: '760px',
+  SINGLE_COLUMN: '650px',
+  MOBILE: '450px',
+};
+
+const HomepageBanner = () => {
+  return (
+    <div
+      css={css`
+        max-height: 423px;
+        position: relative;
+        overflow: hidden;
+        border-radius: 0.5rem;
+        padding-top: calc(423 / 1252 * 100%);
+        background: var(--color-brand-700);
+
+        @media screen and (max-width: ${BREAKPOINTS.SINGLE_COLUMN}) {
+          padding-top: calc(450 / 380 * 100%);
```

Comment:
We might want to leave a comment on what's going on here (or abstract the aspect ratio into a constant at the top of this file).